src/convnets/unet.jl

```diff
@@ -114,7 +117,7 @@ end
 
 function UNet(imsize::Dims{2} = (256, 256), inchannels::Integer = 3, outplanes::Integer = 3,
               encoder_backbone = Metalhead.backbone(DenseNet(121)); pretrain::Bool = false)
-    layers = unet(encoder_backbone, (imsize..., inchannels, 1), outplanes)
+    layers = unet(encoder_backbone, imsize, inchannels, outplanes)
```

Comment:
`inchannels` should somehow be passed in to the encoder backbone here. Of course, we will have to decide how to deal with this in case the user passes in a model with this initialised and also separately `inchannels`